### PR TITLE
docs: add links and updates to whats new 7.0

### PR DESCRIPTION
**Pull request by daniellee**
We have merged some more docs PR's that can be linked to the What's new article. Also updates some of the text for the Panel Edit feature and transformations feature and adds a paragraph about Azure Log Analytics.

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 1]
```diff
@@ -50,12 +56,18 @@ In the future we will add more workflows and integrations so that correlating be
...
 
-Transformations and maths across queries. The data you want to visualize can come from many different places and it is usually not in exactly the right form. The new transformations feature allows you to rename fields, join separate time series together and more - data munging. Usually this requires writing code but this new feature lets you do it in the Grafana UI instead. It also lets you do maths across queries. Lots of data sources do not support this natively so being able to do it in Grafana is a killer feature. For users, with large dashboards or with heavy queries, being able to reuse the query result from one panel in another panel can be a huge performance gain.
+For users, with large dashboards or with heavy queries, being able to reuse the query result from one panel in another panel can be a huge performance gain for slow queries (log or sql queries). From the data source menu in the query editor, you can choose the `--dashboard--` option and then choose the query result from another panel on the same dashboard.
```
Comment (bergquist) [inline docs/sources/guides/whats-new-in-v7-0.md:65]: Is this documented somewhere else? We should link to that. A very powerful feature that's easy to miss.

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 2]
```diff
@@ -50,12 +56,18 @@ In the future we will add more workflows and integrations so that correlating be
...
 
 The [Google Sheets data source](https://grafana.com/grafana/plugins/grafana-googlesheets-datasource) that was published a few weeks ago works really well together with the transformations feature.
 
+We are also introducing a new shared data model for both time series and table data that we call [DataFrame]({{< relref "../developers/plugins/data-frames/#data-frames" >}}). A DataFrame is like a table with columns but we refer to columns as fields. A time series is simply a DataFrame with two fields (time & value).
```
Comment (bergquist) [inline docs/sources/guides/whats-new-in-v7-0.md:69]:
```suggestion
We are also introducing a new shared data model for both time series and table data that we call [DataFrame]({{< relref "../developers/plugins/data-frames/#data-frames" >}}). A DataFrame is like a table with columns but we refer to columns as fields. A time series is a DataFrame with two fields (time & value).
```

**Review by bergquist — APPROVED** [review]
LGTM. Two comments but neither are blockers

File: docs/sources/guides/whats-new-in-v7-0.md (hunk 1 above)
Comment (jessabe) [inline docs/sources/guides/whats-new-in-v7-0.md:65]:
```suggestion
For users with large dashboards or with heavy queries, being able to reuse the query result from one panel in another panel can be a huge performance gain for slow queries (e.g log or sql queries). From the data source menu in the query editor, you can choose the `--dashboard--` option and then choose the query result from another panel on the same dashboard.
```

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 3]
```diff
@@ -108,7 +120,7 @@ Grafana 7.0 adds logging support to one of our most popular cloud provider data
 
 ## Plugins platform
 
-The platform for plugins has been completely re-imagined and provides ready-made components and tooling to help both inexperienced and experienced developers get up and running more quickly. The tooling, documentation and new components will improve plugin quality and reduce long-term maintenance. We are seeing already that a a high quality plugin with the Grafana look and feel can be written in much fewer lines of code than previously.
+The [platform for plugins]({{< relref "../developers/plugins/" >}}) has been completely re-imagined and provides ready-made components and tooling to help both inexperienced and experienced developers get up and running more quickly. The tooling, documentation and new components will improve plugin quality and reduce long-term maintenance. We are seeing already that a a high quality plugin with the Grafana look and feel can be written in much fewer lines of code than previously.
```
Comment (jessabe) [inline docs/sources/guides/whats-new-in-v7-0.md:123]:
```suggestion
The [platform for plugins]({{< relref "../developers/plugins/" >}}) has been completely re-imagined and provides ready-made components and tooling to help both inexperienced and experienced developers get up and running more quickly. The tooling, documentation, and new components will improve plugin quality and reduce long-term maintenance. We are already seeing that a high quality plugin with the Grafana look and feel can be written in much fewer lines of code than previously.
```

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 4]
```diff
@@ -22,21 +22,27 @@ The Grafana 7.0 stable release is scheduled for the 18th of May. In the meantime
...
 - [**Time zones** Time zone support]({{< relref "#time-zone-support" >}})
 
-## New panel editor
+## New panel editor and unified data model
 
-We redesigned the UI for editing panels. We separated panel display settings to a right-hand side pane that you can collapse or expand depending on what your focus is on. With this change we are also introducing our new unified option model and UI for defining data configuration and display options. This unified data configuration system powers a consistent UI for setting data options across visualizations and making all data display settings data-driven and overridable.
+We have redesigned the UI for editing panels. The first visible change is that we have separated panel display settings to a right hand side pane that you can collapse or expand depending on what your focus is on. With this change we are also introducing our new unified option model & UI for defining data configuration and display options. This unified data configuration system powers a consistent UI for setting data options across visualizations as well as making all data display settings data driven and overridable.
```
Comment (jessabe) [inline docs/sources/guides/whats-new-in-v7-0.md:39]:
```suggestion
We have redesigned the UI for editing panels. The first visible change is that we have separated panel display settings to a right-hand side pane that you can collapse or expand depending on what your focus is on. With this change we are also introducing our new unified option model and UI for defining data configuration and display options. This unified data configuration system powers a consistent UI for setting data options across visualizations, as well as making all data display settings data driven and overridable.
```

File: docs/sources/guides/whats-new-in-v7-0.md [hunk 5]
```diff
@@ -50,12 +56,18 @@ In the future we will add more workflows and integrations so that correlating be
 
 ## Transformations
 
-Not just visualizing data from anywhere, in Grafana 7.0 you can transform them, too. By chaining a simple set of point and click transformations users will be able join, pivot, filter, re-name and calculate to get the results they need. Perfect for operations across queries or data sources missing essential data transformations.
+The data you want to visualize can come from many different places and it is usually not in exactly the right form. Users can now transform non-time series data into tables (e.g., JSON files or even simple lookup tables) in seconds without any customization or additional overhead. They can then combine non-time series data with any other data in Grafana; data from an external database or a panel that already exists in one of their current dashboards.
```
Comment (jessabe) [inline docs/sources/guides/whats-new-in-v7-0.md:59]:
```suggestion
The data you want to visualize can come from many different places and it is usually not in exactly the right form. Users can now transform non-time series data into tables (e.g. JSON files or even simple lookup tables) in seconds without any customization or additional overhead. They can then combine non-time series data with any other data in Grafana; an example would be for users to take data from an external database, and data from a panel that already exists in one of their current dashboards.
```

File: docs/sources/guides/whats-new-in-v7-0.md (hunk 1 above)
Comment (daniellee) [reply docs/sources/guides/whats-new-in-v7-0.md]: Doesn't seem so. Will follow up

**Comment by marefr** [comment]
Going to have a look at backend plugins sections/add links.

**Pull request merged**